src/main/java/io/pravega/connector/boomi/PravegaUtil.java

```diff
@@ -90,21 +90,14 @@ static void testConnection(ConnectorContext connectorContext, String keycloakJSO
             // check connection and scope
             try {
                 streamManager.listStreams(pravegaConfig.getScope()).hasNext();
-            } catch (Throwable t) {
-                // peel exception from Future
```

Comment:
why was this line removed?  if we don't peel the exception, we might not see that NoSuchScopeException was the cause